diagnostic.md

```diff
@@ -94,4 +94,6 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+'teamMember'['hobby']' is 'exercise'
+'teamMember['name']'? is an error because you do not specify if you are referring to the "given" or "surname"
```

Comment:
The value of teamMember['name'] is actually an object, { given: 'Jason', surname: 'Weeks' }.